site/docs/configuration.md

```diff
@@ -70,6 +70,7 @@ Iceberg tables support table properties to configure table behavior, like the de
 | commit.manifest-merge.enabled      | true             | Controls whether to automatically merge manifests on writes   |
 | history.expire.max-snapshot-age-ms | 432000000 (5 days) | Default max age of snapshots to keep while expiring snapshots    |
 | history.expire.min-snapshots-to-keep | 1                | Default min number of snapshots to keep while expiring snapshots |
+| history.expire.max-ref-age-ms      | forever            | For snapshot references except the `main` branch, default max age of snapshot references to keep while expiring snapshots. The `main` branch never expires. |
```

Comment:
Should this be something like `unset (forever)` or `Long.MAX_VALUE (forever)`?

Reply:
Because this will be used as the default if the value is not set in ref, I am making it `Long.MAX_VALUE` to avoid an additional null check.